lbrynet/dht/node.py

```diff
@@ -216,14 +219,10 @@ def getPeersForBlob(self, blob_hash):
         if result:
             if blob_hash in result:
                 for peer in result[blob_hash]:
-                    if self.node_id != peer[6:]:
-                        host = ".".join([str(ord(d)) for d in peer[:4]])
-                        if host == "127.0.0.1" and "from_peer" in result \
-                                and result["from_peer"] != "self":
```

Comment:
This is coming from https://github.com/lbryio/lbry/blob/0b771c16ba058d3919b2bac482f4d725473a72fa/lbrynet/dht/node.py#L369

Reply:
see discussion at end here: #968